agent/pkg/tunnel/config/config.go

```diff
@@ -21,6 +21,10 @@ type TunnelAgentConfig struct {
 	// ListenPort indicates the listen port of tunnel agent
 	// default 10006
 	ListenPort int `json:"listenPort"`
+	// IsGatewayEnable indicates whether module gateway is enabled
+	// if gateway is enable, tunnel agent only have outbound traffic
+	// default false
+	IsGatewayEnable bool `json:"isGatewayEnable"`
```

Comment:
```suggestion
	GatewayEnable bool `json:"gatewayEnabled"`
```

Reply:
ok, I decide to delete this element, this value is equal to module gateway enable or not